save_selected_images.py

```diff
@@ -0,0 +1,51 @@
+"""
+    This script shows one by one each image within a folder, by natural order,
+    and save selected ones typing "s" key in "selected" folder
+"""
+
+
+import cv2
+import glob
+import os
+from natsort import natsorted
+import argparse
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    '-images_path',
+    type=str,
+    help='path to folder with images to be detected')
+parser.add_argument(
+    '--scale',
+    type=int,
+    default=3,
+    help='Downscale the image for better visualization. Doesnt change saved image')
+
+
+def main():
+    args = parser.parse_args()
+    os.chdir(args.images_path)
+    os.mkdir("selected")
```

Comment:
This crashes if directory "selected" already exists. This could happen if there is a big dataset and you select some pics (folder is created as it is the first time),  and then come back to it later, folder already exists. 

I suggest the following:

```python
out_folder = "selected"
    if not os.path.exists(out_folder):
        os.mkdir(out_folder)

```

Reply:
good way